schema-bridge.graphql

```diff
@@ -1,5 +1,5 @@
 # Subgraph Schema: Bridge
-# Version: 1.0.0
+# Version: 1.0.1
```

Comment:
Well, technically it should be major, since it is a breaking change. Old code won’t work with this schema change. But felt too premature to make it major 😅 

Can definitely do minor though 👍